### PR TITLE
{vis}[foss/2016a] PyQt 4.11.4 w/ Python 2.7.11

### PR DESCRIPTION
**Pull request by ysagon**

### CODE REVIEW

File: easybuild/easyconfigs/p/PyQt/PyQt-4.11.4-foss-2016a-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+sources = ['%(name)s-x11-gpl-%(version)s.tar.gz']
+source_urls = ['http://sourceforge.net/projects/pyqt/files/PyQt4/PyQt-%(version)s']
+
+python = 'Python'
+pyver = '2.7.11'
+pyshortver = '.'.join(pyver.split('.')[:2])
+versionsuffix = '-%s-%s' % (python, pyver)
```
Comment (boegel) [inline easybuild/easyconfigs/p/PyQt/PyQt-4.11.4-foss-2016a-Python-2.7.11.eb]: @ysagon please use `%(pyver)s`, and move up `versionsuffix = ...` right below `version = ...`

```python
versionsuffix = '-Python-%(pyver)s'
...
dependencies = [
    ('Python', '2.7.11'),
...
```

and then drop the `python`, `pyver` and `pyshortver` local variables entirely...

**Comment by boegel** [comment]
@ysagon please merge https://github.com/ysagon/easybuild-easyconfigs/pull/1 to fix a conflict with current `develop`, maybe you deleted this easyconfig in a previous PR that got merged already?

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2418.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/baac3f156bd5d137401daa5b08808e1a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2136.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/960a1bebd953b7ee34cc9b6336d5d743 for a full test report.

**Comment by boegel** [comment]
@ysagon missing dep?

```
make[1]: *** [sipphononPhononMediaController.o] Error 1
/tmp/vsc40023/easybuild_build/PyQt/4.11.4/foss-2016a-Python-2.7.11/PyQt-x11-gpl-4.11.4/sip/phonon/abstractaudiooutput.sip:28:33: fatal error: abstractaudiooutput.h: No such file or directory
 #include <abstractaudiooutput.h>
```

**Comment by boegel** [comment]
@ysagon ping?

**Comment by migueldiascosta** [comment]
@ysagon, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**